src/tree/tree_model.cc

```diff
@@ -733,7 +767,9 @@ XGBOOST_REGISTER_TREE_IO(GraphvizGenerator, "dot")
 constexpr bst_node_t RegTree::kRoot;
 
 std::string RegTree::DumpModel(const FeatureMap& fmap, bool with_stats, std::string format) const {
-  CHECK(!IsMultiTarget());
+  if (format != "dot") {
+    MTNotImplemented();
+  }
```

Comment:
```suggestion
  CHECK_EQ(format, "dot") << "Dump format " << format << MTNotImplemented();
```

Reply:
No, we don't require the check for scalar trees

Reply:
Fixed the error message.